packages/cli/src/validators/__tests__/no-url.validator.test.ts

```diff
@@ -0,0 +1,26 @@
+import { NoUrl } from '../no-url.validator';
+import { validate } from 'class-validator';
+
+describe('NoUrl', () => {
+	class Entity {
+		@NoUrl()
+		name = '';
+	}
+
+	const entity = new Entity();
+
+	describe('URLs', () => {
+		const URLS = ['http://google.com', 'www.domain.tld'];
+
+		for (const str of URLS) {
+			test(`should block ${str}`, async () => {
+				entity.name = str;
+				const errors = await validate(entity);
+				expect(errors).toHaveLength(1);
+				const [error] = errors;
+				expect(error.property).toEqual('name');
+				expect(error.constraints).toEqual({ NoUrl: 'Potentially malicious string' });
+			});
+		}
```

Comment:
[`test.each`](https://jestjs.io/docs/api#testeachtablename-fn-timeout) is made for cases like this:

```suggestion
  test.each(['http://google.com', 'www.domain.tld'])(
    'should block %s', async (str) => {
				entity.name = str;
				const errors = await validate(entity);
				expect(errors).toHaveLength(1);
				const [error] = errors;
				expect(error.property).toEqual('name');
				expect(error.constraints).toEqual({ NoUrl: 'Potentially malicious string' });
        }
  )

```

Reply:
Thanks! I've always found a simple loop more readable and easier to remember, for `test.each` I always have to look up the syntax 😓 Is `test.each` faster or better in some other way?

Reply:
Fair point. Personally I find `test.each` more readable :D There are subtle differences also, like mentioned in this [SO comment](https://stackoverflow.com/a/65908574)